### PR TITLE
feat(auth): add VAT tax population script

### LINKED ISSUE #9359: Script to update existing EU subscribers to have appropriate tax rate applied
For existing customers, we need to ensure their subscription has the appropriate tax rate applied if they are paying in Euro's. The script should iterate over all subscriptions in an active state that are in the Euro currency, and ensure they have the appropriate tax rate linked.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-3652)

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* We want existing subscribers to have VAT included on their invoices.

This commit:

* Adds the appropriate VAT to the subscription items for existing
  subscribers that didn't have a tax rate listed.

Closes #9359

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import { ACTIVE_SUBSCRIPTION_STATUSES } from 'fxa-shared/subscriptions/stripe';
```
Comment (chenba) [inline packages/fxa-auth-server/scripts/populate-vat-taxes.ts:5]: I'm uncertain if this import is going to work.  Maybe change to a relative import?

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        // Skip if any subscription items already has a tax rate
+        if (
+          sub.items.data.some(
+            (item) => item.tax_rates && item.tax_rates.length > 0
+          )
+        ) {
+          continue;
+        }
+
+        // Skip if the customer is not paying in Euro
```
Comment (chenba) [inline packages/fxa-auth-server/scripts/populate-vat-taxes.ts:41]: This comment doesn't seem to match the code that immediately follows?

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  // Locate all plans using a VAT currency
+  const plans = (await stripe.plans.list({ limit: 100 })).data.filter((plan) =>
+    ['eur', 'chf'].includes(plan.currency)
+  );
+
+  for (const activeStatus of ACTIVE_SUBSCRIPTION_STATUSES) {
+    for (const plan of plans) {
+      for await (const sub of stripe.subscriptions.list({
+        limit: 100,
```
Comment (chenba) [inline packages/fxa-auth-server/scripts/populate-vat-taxes.ts:28]: Why only 100 subscriptions per plan?

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts (hunk 1 above)
Comment (bbangert) [reply packages/fxa-auth-server/scripts/populate-vat-taxes.ts:5]: It seemed to work every time I ran it.

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts (hunk 3 above)
Comment (bbangert) [reply packages/fxa-auth-server/scripts/populate-vat-taxes.ts:28]: This is an async iterator, 100 is the max per 'page' of results per async iteration.

**Review by chenba — APPROVED** [review]
LGTM

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts (hunk 3 above)
Comment (chenba) [reply packages/fxa-auth-server/scripts/populate-vat-taxes.ts:28]: Not sure what it is about this interface that keep tripping me up.  I made a similar comment in a previous SubPlat script PR.  🙄

File: packages/fxa-auth-server/scripts/populate-vat-taxes.ts (hunk 2 above)
Comment (bbangert) [reply packages/fxa-auth-server/scripts/populate-vat-taxes.ts]: Fixed, thanks!

**Pull request merged**